### PR TITLE
MIlestone 3: Portfolio Mobile Version.

### PR DESCRIPTION
**Pull request by Leon-Espitia**
## Mobile Version: Contact Form 👨‍💻 
In this Milestone I:
- Added the final section on the Portfolio Mobile Version (Contact Form).
- Apply remote pair programming so I got contributions from my coding partner. (Mustafa Fahimy). 🤝

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -222,7 +222,28 @@ <h2 class="title">About<br>Myself</h2>
...
+                        are interested in developing, 
+                        a feature that you need built or a
+                        project that needs coding. I'd
+                        love to help with it 
+                    </p>
+                </div>                
+                <form action="https://formspree.io/f/xqknnnpl" method="post" class="form_area_contact">
+                   
+                    <input type="text" name="name" id="name_contact" maxlength="30" placeholder="Full Name" required>
+                    <input type="text" id="email_contact" placeholder="Email" required>
```
Comment (vechicin) [inline index.html]: According to the project requirements, this `<input>` should be of `type="email"`.

Screen Shot 2022-05-31 at 16 28 40

Kindly fix this.

File: style.css [hunk 2]
```diff
@@ -253,3 +253,80 @@ h2 {
...
+/* contact form start here */
+.contac_me {
+  background-color: #6070ff;
+  width: 375px;
+  border-top-left-radius: 70px;
+  padding-bottom: 10px;
+  margin-top: -10px;
+  background-image: url("./assets/backgrounds/header-shapes\ mobile.png");
+  background-repeat: no-repeat;
+}
```
Comment (vechicin) [inline style.css]: Your design doesn't look quite like the template design.

Here's your design:

Screen Shot 2022-05-31 at 16 31 28

Here's the template design:

Screen Shot 2022-05-31 at 16 32 13
(I apologize about the colored borders, I had to take the screenshot while other students were working on the template.)

Do you see how different they are? Kindly fix your design as it is a project requirement.

Additionally, there is now a strange margin to the right of the other sections' layout and to the left of the contact form. Kindly fix that as well.

Screen Shot 2022-05-31 at 16 34 04

File: style.css [hunk 3]
```diff
@@ -253,3 +253,80 @@ h2 {
...
+  background-color: #fff;
+  color: #6070ff;
+  width: fit-content;
+  font-weight: 500;
+  border-radius: 5px;
+  padding: 5%;
+  cursor: pointer;
+  padding-top: 15px;
+  margin-top: 10px;
+}
```
Comment (vechicin) [inline style.css]: According to the project requirements, this button should have styling for several interactions and those stylings should follow the template design. You are missing a couple of stylings and the ones you have don't follow the template design. Kindly fix this.

Screen Shot 2022-05-31 at 16 35 56

Screen Shot 2022-05-31 at 16 36 16

**Review by vechicin — CHANGES_REQUESTED** [review]
# Hi @Leon-Espitia  ✋, 

Good job so far! 🔥 
There are some issues that you still need to work on to go to the next project but you are almost there! 🚨 
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights ⚡ 

- [x] Fantastic PR title/description
- [x] Good commit history
- [x] Correct GitHub Flow

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions 📂 
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css [hunk 4]
```diff
@@ -253,3 +256,96 @@ h2 {
...
+#submit_btn:disabled {
+  background: #fff;
+  color: rgb(71, 72, 75);
+  border: 1px solid #888991;
+}
+
+#submit_btn:enabled {
+  background: #fff;
+  color: #6070ff;
+}
```
Comment (Kakalanp) [inline style.css]: Kindly remove these lines, as they're interfering with the button effect :mag: 

Whenever I remove those from my end, the colors work ideally :thinking:

**Review by Kakalanp — CHANGES_REQUESTED** [review]
# STATUS: Changes required :exclamation: 

### Hello there @Leon-Espitia ! hi

### Great work on making the changes requested by a previous reviewer 👏
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet. :memo: 

almost there

## Highlights :+1: 
- No linter errors :heavy_check_mark:
- Repo looks professional :briefcase:
- Solid pull request :scroll:
- Good UI design :monocle_face: 

 ## Required Changes ♻️
***Check the comments under the review.***

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Kakalanp — APPROVED** [review]
# STATUS: APPROVED :green_circle: 

### Your project is approved @Leon-Espitia! :tada: 

There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! :clinking_glasses: 

almost there

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**